cmd/etcd-manager/main.go

```diff
@@ -203,6 +209,11 @@ func RunEtcdManager(o *EtcdManagerOptions) error {
 	var discoveryProvider discovery.Interface
 	var myPeerId privateapi.PeerId
 
+	// start etcd-manager metrics if the etcd manager metrics port is defined
+	if o.EtcdManagerMetricsPort != 0 {
```

Comment:
Why aren't we using a default port and add it to the well known ones in kOps? I think those metrics here are quite interesting for everyone using etcd-manager.

Reply:
this means that metrics port is not enabled by default. We cannot enable this by default before kops has configurations in place. Why? Because it has fatal logging level and we do have always 2 etcd-manager clusters in kops clusters. Which means that if we define default port we are going to have 2 clusters trying to bind to same port and with fatal log level -> it will fail always. That is why we need define this setting in kops side instead of etcd-manager

Reply:
OK, fair enough

Reply:
Thanks for clarification